Dockerfile.builder

```
@@ -8,7 +8,7 @@
 # You can test the image using `./builder-run.sh`. For instance:
 #   $ ./builder-run.sh ./build-backend.sh
 
-FROM golang:1.20-bullseye
+FROM golang:1.22
```

```suggestion
FROM golang:1.22-bookworm
```

We probably want bookworm or bulleyes since we install other dependencies. From the README:

https://hub.docker.com/_/golang

> `golang:<version>`
> This is the defacto image. If you are unsure about what your needs are, you probably want to use this one. It is designed to be used both as a throw away container (mount your source code and start the container to start your app), as well as the base to build other images off of.
> 
> Some of these tags may have names like bookworm or bullseye in them. These are the suite code names for releases of [Debian⁠](https://wiki.debian.org/DebianReleases) and indicate which release the image is based on. If your image needs to install any additional packages beyond what comes with the image, you'll likely want to specify one of these explicitly to minimize breakage when there are new releases of Debian.